.github/workflows/verify.yml

```
@@ -0,0 +1,74 @@
+name: Verify Submodule Pull Request
+
+on:
+  workflow_call:
+    inputs:
+      repo:
+        required: false
+        type: string
+        default: Lombiq/Open-Source-Orchard-Core-Extensions
+
+env:
+  PULL_REQUEST_TITLE: ${{ github.event.pull_request.title }}
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  parent-pull-request-exists:
+    runs-on: ubuntu-latest
+    name: Parent PR Exists
+    steps:
+      - name: Print PR Title
+        run: echo $Env:PULL_REQUEST_TITLE
+
+      - name: Add Scripts
+        run: |
+          $code = @'
+          <#
+          .Synopsis
+          Fails the current step with the provided message. Same as "core.setFailed()" from the official toolkit
+          (https://github.com/actions/toolkit).
+          #>
+          function Set-Failed([Parameter(Mandatory=$True)] [string] $Message)
+          {
+              echo "::error::$Message"
+              exit 1
+          }
+
+          <#
+          .Synopsis
+          Returns $True if any item is coming from the pipe.
+          #>
+          function Any-Object([Parameter(Mandatory=$True, ValueFromPipeline = $True)] $Input)
+          {
+              foreach($item in $Input)
+              {
+                  return $True
+              }
+
+              return $False
+          }
+          '@
+
+          $ProfileDirectory = [System.IO.Path]::GetDirectoryName($Profile)
+          [System.IO.Directory]::CreateDirectory($ProfileDirectory)
+
+          echo $code >> $Profile
+
+      - name: Current PR Title Includes Issue Code
+        run: |
+          if ($Env:PULL_REQUEST_TITLE -match '^\s*\w+-\d+\s*:') { exit 0 }
+          Set-Failed 'The pull request is not in the right format. Please start with the issue code followed by a colon e.g. "PROJ-123: My PR Title".'
+
+      - name: Search Parent Repository for Matching PR
+        run: |
+          $issueCode = $Env:PULL_REQUEST_TITLE -replace '^\s*(\w+-\d+)\s*:.*$', '$1'
+
+          $url = 'https://api.github.com/repos/Lombiq/Open-Source-Orchard-Core-Extensions/pulls?state=open&per_page=100'
+          $titles = curl -H 'Accept: application/vnd.github.v3+json' $url | ConvertFrom-Json | % { $_.title }
+
+          $lookFor = "${issueCode}:"
+          if ($titles | ? { $_ -and $_.Contains($lookFor) } | Any-Object) { exit 0 }
```

If we want the PR title to start with the issue code (ignoring whitespace), then let's express that in code:
```suggestion
          if ($titles | ? { $_ -and $_.Trim().StartsWith($lookFor) } | Any-Object) { exit 0 }
```